core/storage/activity/gae_models.py

```diff
@@ -44,28 +44,25 @@ class ActivityReferencesModel(base_models.BaseModel):
     activity_references = datastore_services.JsonProperty(repeated=True)
 
     @staticmethod
-    def get_deletion_policy():
-        # type: () -> base_models.DELETION_POLICY
+    def get_deletion_policy() -> base_models.DELETION_POLICY:
         """Model doesn't contain any data directly corresponding to a user."""
         return base_models.DELETION_POLICY.NOT_APPLICABLE
 
     @staticmethod
-    def get_model_association_to_user():
-        # type: () -> base_models.MODEL_ASSOCIATION_TO_USER
+    def get_model_association_to_user() -> (
+            base_models.MODEL_ASSOCIATION_TO_USER):
```

Comment:
In some other place @hardikkat24 did
```suggestion
    def get_model_association_to_user(
    ) -> base_models.MODEL_ASSOCIATION_TO_USER:
```

or

```suggestion
    def get_model_association_to_user(
    	) -> base_models.MODEL_ASSOCIATION_TO_USER:
```

not sure which is better.

Reply:
First one seems better. Reason: If the function had arguments, the placement of the bracket will still be same.

Reply:
Used the first suggestion.